### PR TITLE
Unify save/cancel button styles.

### PR DESCRIPTION
**Pull request by akash1810**
Adding labels to the save and cancel buttons of the usage rights editor form.

Also removing some duplicated css :tada:.

button

### CODE REVIEW

**Comment by paperboyo** [comment]
is it me or are the icons not aligned correctly to text (too low)?

**Comment by theefer** [comment]
:+

**Comment by alastairjardine** [comment]
@paperboyo Good spot. @akash1810 Could you fix? Also, could we put the save on the right and the cancel on the left please? Thanks!

**Comment by akash1810** [comment]
@paperboyo you have ridiculously good sight!

screen shot 2015-08-20 at 12 36 07

> the save on the right and the cancel on the left please

Just on usage, or on metadata edits too?

**Comment by theefer** [comment]
+1 to swap and have Save on the right, seems more standard.

**Comment by alastairjardine** [comment]
@akash1810 Meta edits & usage please – makes sense as it's easier for the eye to lock on to. Reduces cognitive load just a little. Ta!

**Comment by akash1810** [comment]
screen shot 2015-08-20 at 13 37 25

screen shot 2015-08-20 at 13 37 32

File: kahuna/public/js/forms/gr-xeditable/gr-xeditable.js [hunk 1]
```diff
@@ -18,6 +15,10 @@ grXeditable.run(['editableOptions', 'editableThemes', function (editableOptions,
      */
     editableOptions.theme = 'default';
 
-    editableThemes['default'].submitTpl = submitButton;
-    editableThemes['default'].cancelTpl = cancelButton;
+    // override the default template for submit and cancel buttons because we re-order the buttons
+    // https://github.com/vitalets/angular-xeditable/blob/master/src/js/themes.js
```
Comment (akash1810) [inline kahuna/public/js/forms/gr-xeditable/gr-xeditable.js]: :neutral_face:

**Comment by theefer** [comment]
:+1:

**Pull request merged**